MsmqReturnToSourceQueue/MsmqAddress.cs

```diff
@@ -2,7 +2,7 @@
 using System.Net;
 
 
-internal struct MsmqAddress
+public struct MsmqAddress
```

Comment:
why did this need to change?  The entire project should be able to reference something that's `internal`

Reply:
Tests

Reply:
in the interest of contributing this back, why don't we do this instead:
https://msdn.microsoft.com/en-us/library/system.runtime.compilerservices.internalsvisibletoattribute(v=vs.110).aspx

That way we don't have to change access modifiers.  Normally I'd say nbd, but i think we'll get gruff for this too if we want to contribute back to nsb.

Reply:
I think it doesn't matter - this is deprecated and not really in use.